season-config.html

```diff
@@ -15,24 +15,31 @@ <h3 id="newSeasonConfirmation"></h3><br><br>
 <br><br><br><br>
 <h3 id = "seasonQuestionsTitle" style = "text-align: center"></h3>
 
-<h3>Enter in any scoring opportunities in the first text box
-    and how many points that scoring opportunity is
-    worth in the second text box. <br><br>
-    Please enter only phrases for the element name and
-    integers for the points.<br><br>
+<h3>Enter in any scoring opportunities in the first text box and how many points that scoring opportunity is worth in the second text box.
+    <br><br>
+    Also, please enter the type of answer to be expected when answering questions about the scoring opportunity in 
```

Comment:
Why do we need the "points worth"?

Reply:
I just left it there at the start in case someone wanted to improve the app and make a feature like team 1 scores 20 pts per average.